### ISSUE TITLE
Keywords with "-"

### ISSUE THREAD

**Issue by manocarayannis** [Donatello-za/rake-php-plus#9]
Words like "zinc-plated" or "gold-plated" should stay together. 

for example, they query "Everbilt 3 in. Zinc-Plated Corner Brace (4-Pack)" I get:

["plated corner brace","everbilt 3","zinc","pack"]

The first, should be "zinc-plated",  then probably "corner  brace" etc..

**Comment by Donatello-za** [comment]
I agree with you. There is a number of issues that I'll have to find a solution for with hyphenated words, see: https://github.com/Donatello-za/rake-php-plus/issues/5

**Comment by Donatello-za** [comment]
If I incorporate better handling of hyphenated words, I can get it to provide the following results:

```php
$text = "Everbilt 3 in. Zinc-Plated Corner Brace (4-Pack)";
$phrases = RakePlus::create($text)->get();
print_r($phrases);
```
Will Return:
```
Array
(
    [0] => everbilt 3
    [1] => zinc-plated corner brace
    [2] => 4-pack
)
```

or for keywords:

```php
$text = "Everbilt 3 in. Zinc-Plated Corner Brace (4-Pack)";
$phrases = RakePlus::create($text)->keywords();
print_r($phrases);
```
Which will give you:
```
Array
(
    [0] => everbilt
    [1] => 3
    [2] => zinc-plated
    [3] => corner
    [4] => brace
    [5] => 4-pack
)
```

Would you be happy with that? If so I can add it to my next release.

**Comment by manocarayannis** [comment]
Yes that would be the expected result. Thank you!

**Comment by Donatello-za** [comment]
Fixed in v1.0.7 as per commit: https://github.com/Donatello-za/rake-php-plus/commit/97d166b02dbed0c3721f18f2f305d19c820c4ef6
